staging/src/k8s.io/apimachinery/Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/apimachinery",
-	"GoVersion": "go1.9",
```

Comment:
Need to revert this change.

Reply:
reverted all the godep updates. Only ran  `./hack/update-bazel.sh`